ui/index.html

```diff
@@ -26,7 +26,7 @@
             <div id="index-page-container" class="row"><!-- --></div>
             <div id="index-page-template"><!--
                 {macro renderSearchBlock()}
-                    <form class="index-search-form" role="search" aria-label="__MSG__SEARCH_FOR_CONTENT_FOLDERS_PEOPLE_GROUPS__">
+                    <form action="." class="index-search-form" role="search" aria-label="__MSG__SEARCH_FOR_CONTENT_FOLDERS_PEOPLE_GROUPS__">
```

Comment:
When I try this one on the iOS simulator, it shows "Go" instead of "Search". It's working correctly for the others though. Any idea what's causing this?

![screen shot 2015-01-29 at 06 52 06](https://cloud.githubusercontent.com/assets/109850/5959407/e6e41cf8-a782-11e4-96a4-c6a1cba396d1.png)